perf/lua/luafun.lua

```diff
@@ -0,0 +1,77 @@
+local fun = require("fun")
+local clock = require("clock")
+local t = require("tarantool")
+local benchmark = require("benchmark")
+
+local USAGE = [[
+ This benchmark measures the performance of luafun module.
+]]
+
+local params = benchmark.argparse(arg, {}, USAGE)
+
+local bench = benchmark.new(params)
+
+local _, _, build_type = string.match(t.build.target, "^(.+)-(.+)-(.+)$")
+if build_type == "Debug" then
+    print("WARNING: tarantool has built with enabled debug mode")
+end
+
+--
+-- Data for `drop_while` benchmark.
+--
+
+local DROP_WHILE_CYCLES = 10^3
+local DROP_WHILE_N = 10^7
+
+local function DROP_WHILE_FILTER(x)
+    return x <= 50
+end
+
+local DROP_WHILE_DATA = {}
+for _ = 1, DROP_WHILE_N / 2 do
+    local value = math.random(50)
```

Comment:
Since we use `math.random()` it is better to use `math.randomseed(0)` (not `0` neccessary) to avoid different results per run. We don't know the state of the `tarantool` after startup. May be it's already called `math.random()` several times.